tests/ops/conv_2d_nhwc_fhwc_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+
+import torch
+
+from iree.turbine.ops.conv_fwd import conv_2d_nhwc_fhwc
+
+N = 2
+H = 16
+W = 16
+C = 3
+F = 1
+Hk = 3
+Wk = 3
+
+strides = [
+    [1, 1],
+    [2, 2],
+]
+
+dilations = [
+    [1, 1],
+    [2, 2],
+]
+
+dtype = torch.float32
+
+device = torch.device("cuda:0") if torch.cuda.is_available() else None
+
+
+@pytest.mark.parametrize("s", strides)
+@pytest.mark.parametrize("d", dilations)
+def testCustomConvImplementationEager(s, d):
+    gen = torch.Generator(device=device)
+    gen.manual_seed(10)
+    x = torch.randn([N, H, W, C], generator=gen, dtype=torch.float32, device=device)
```

Comment:
if you decide to take the output dtype suggestion, then would be good to add tests for different types, bf16, I32 perhaps?